nss-day-cohort-48/fun-website-1-the-single-coders
# Updated Photos and Linking in Navbar

I still need to add a link to AMD page.

##Steps to Test

1. Pull this branch
2. Open in Chrome
3. Ensure Photos are present and Navbar goes to Latest Articles and NVDIA page
